src/transformers/tokenization_utils_tokenizers.py

```diff
@@ -1089,7 +1090,12 @@ def is_base_mistral(model_id: str) -> bool:
                     return True
             return False
 
-        if pretrained_model_name_or_path is not None and (is_local or is_base_mistral(pretrained_model_name_or_path)):
+        if is_offline_mode():
+            is_local = True
+
+        if pretrained_model_name_or_path is not None and (
+            is_local or (not is_local and is_base_mistral(pretrained_model_name_or_path))
+        ):
```

Comment:
@ArthurZucker Can we get a backport of this fix, specifically:

```python
if is_offline_mode():
            is_local = True
```

and

```python
if pretrained_model_name_or_path is not None and (
            is_local or (not is_local and is_base_mistral(pretrained_model_name_or_path))
        ):
```

Without this offline loading is broken (also for non-mistral models).